kvbc/src/kvbc_app_filter/kvbc_app_filter.cpp

```diff
@@ -371,12 +371,12 @@ FindGlobalEgIdResult KvbAppFilter::findGlobalEventGroupId(uint64_t external_even
   uint64_t private_start = getValueFromLatestTable(client_id_ + "_oldest");
   uint64_t private_end = getValueFromLatestTable(client_id_ + "_newest");
 
-  if (not private_start) {
+  if (not private_start and not private_end) {
```

Comment:
This is a good find!! Now, the code below might throw :/ I need to think about this a bit more.

Reply:
I added this change as part of #2592

Reply:
Thanks, removed.